boards/nxp/frdm_mcxn947/CMakeLists.txt

```diff
@@ -6,3 +6,10 @@
 
 zephyr_library()
 zephyr_library_sources(board.c)
+
+# Data bus width is used by the SDK driver and processes it as a compile time option
+if(CONFIG_MIPI_DBI_NXP_FLEXIO_LCDIF)
```

Comment:
Can we do this processing in the CMake lists for the driver? I assume this would be needed by targets other than the FRDM board

Reply:
Sure, I can move it and use the label `zephyr_flexio_lcd` that is defined at the board level and is used by the board shields. Would that work?